src/controller/proxy/local.go

```diff
@@ -135,6 +138,27 @@ func (l *localHelper) DeleteManifest(repo, ref string) {
 	}
 }
 
+// DeleteManifestQuiet delete manifest without existing check
+func (l *localHelper) DeleteManifestQuiet(ctx context.Context, repo, ref string) {
+	log.Infof("Remove tag from repo if it is exist, repo: %v ref: %v", repo, ref)
+	// convert tag to digest
+	bCtx := orm.Copy(ctx)
+	opt := &artifact.Option{}
+	art, err := l.artifactCtl.GetByReference(bCtx, repo, ref, opt)
```

Comment:
what about if the art is a index? Shouldn't we handle the references of this art?